docs/javascript/v2/how--to-guides/record-and-live-stream/hls/hls-player.mdx

```diff
@@ -0,0 +1,258 @@
+---
+title: HLS Player (Beta)
+nav: 7.23
+---
+
+The HMSHLSPlayer is an HLS player offered by 100ms that can be used to play HLS streams. The player takes a URL and video element to play the stream. 
+
+## How to integrate HLS Player SDK
+
+You can use Node package manager or yarn to add HMSHLSPlayer sdk to your project.
+Use https://www.npmjs.com/package/@100mslive/hls-player as the package source.
```

Comment:
can we add a quick snippet that can be copy-pasted? `npm i @100mslive/hls-player`